dk-installer.py

```diff
@@ -775,14 +775,18 @@ def execute(self, action, args):
 
 class HelmInstallStep(Step):
     chart_info: tuple[str, str] = None
+    values_arg: str = None
 
     def execute(self, action, args):
         release, chart_ref = self.chart_info
+        values_file = getattr(args, self.values_arg or "", None)
```

Comment:
If you use a default value when declaring the argument instead of using the `getattr` here, it would be more consistent with how it's being done in other places. No biggie =)

Reply:
Could you elaborate on the suggestion?

My thinking was that `values_arg` could be set to different argument names in the subclasses, e.g., in case we want to add a `--services-values` argument in the future for configuring the `ObsHelmInstallServicesStep` step.

Reply:
Applied the suggested change - thanks!